lib/ts/recipe/emailpassword/recipeImplementation.ts

```diff
@@ -20,15 +20,22 @@ export default function getRecipeInterface(
                 email,
                 password,
                 tenantId,
+                shouldAttemptAccountLinkingIfAllowed,
                 userContext,
             }: {
                 email: string;
                 password: string;
                 tenantId: string;
-                userContext: any;
+                shouldAttemptAccountLinkingIfAllowed: boolean;
```

Comment:
this is not needed anymore since we already have createNewRecipeUser function which is like signup without accountt linking

Reply:
Done